### ISSUE TITLE
image response problem

### ISSUE THREAD

**Issue by zione-kushwaha** [babakcode/flutter_gemini#45]
i am not getting response from for image only getting instance of image part 

  Future<void> _processImage() async {
    if (_selectedImage == null) {
      setState(() {
        _response = 'No image selected';
      });
      return;
    }

    try {
      final bytes = await _selectedImage!.readAsBytes();

      Gemini.instance.textAndImage(
        text: 'Analyze this image and describe its content.',
        images: [bytes],
      ).then((response) {
        final parts = response?.content?.parts;
        final lastText = parts ?? 'No text content found';

        setState(() {
          _response = lastText.toString();
        });
      }).catchError((error) {
        setState(() {
          _response = 'Error: $error';
        });
      });
    } catch (e) {
      setState(() {
        _response = 'Error: $e';
      });
    }
  }

i tried using that     final lastText = parts ?? 'No text content found';.......parts.text but also there is error saying in text ........now only i am getting response as instance of textPart.....how to solve this problem

**Comment by ArpitAswal** [comment]
Yes, I am also facing the same issue. The compiler says, the getter 'text' isn't defined for the type 'Part'.

problem link: https://github.com/babakcode/flutter_gemini/issues/45#issue-2731920223
